caf-logging-tomcat-juli/LICENSE_HEADER

```diff
@@ -0,0 +1,9 @@
+Tomcat-Slf4j-Logback (https://github.com/tomcat-slf4j-logback/tomcat9-slf4j-logback/)
+
+Copyright (c) ${license.git.copyrightYears} Tomcat-Slf4j-Logback.
+
+All rights reserved. This program and the accompanying materials are made available under the
+terms of the Eclipse Public License v1.0 which accompanies this distribution, and is available
+at https://www.eclipse.org/legal/epl-v10.html.
+
+Contributors: Tomcat-Slf4j-Logback Team.
```

Comment:
Again I'm struggling to understand the legal implications of what you're proposing in this PR.

Reply:
Please suggest to me how to handle these license files when modifying third-party APIs?
I don't know how to include license files when creating new module with their source.